Makefile

```diff
@@ -434,9 +435,9 @@ catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
 
 # A valid Git branch from https://github.com/openshift/oadp-operator
-PREVIOUS_CHANNEL ?= oadp-1.3
+PREVIOUS_CHANNEL ?= oadp-1.4
```

Comment:
for each branch, we will have a different upgrade test (same foundation, but different DPA and logic, for example). Also, we will always test upgrades from next minor version